MvvmCross/Windows/Uwp/Views/MvxWindowsMultiRegionViewPresenter.cs

```diff
@@ -70,14 +70,14 @@ public override void Close(IMvxViewModel viewModel)
             }
         }
 
-        private static Type GetViewType(MvxViewModelRequest request)
+        protected static Type GetViewType(MvxViewModelRequest request)
         {
             var viewFinder = Mvx.Resolve<IMvxViewsContainer>();
             return viewFinder.GetViewType(request.ViewModelType);
         }
 
         // Implementation from: http://stackoverflow.com/a/1759923/80186
-        internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
+        protected internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
```

Comment:
Yes, that is what I meant. If you subclass and implement a method with the same signature, it will clash. Which probably will not be a huge deal.